### PR TITLE
chore(rxAppRoutes): decompose monolithic source code

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: [FRMW-346](https://jira.rax.io/browse/FRMW-346)

### LGTMS

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxAppRoutes/scripts/rxAppRoutes.js [hunk 1]
```diff
@@ -1,139 +1,4 @@
...
- */
-angular.module('encore.ui.rxAppRoutes', ['encore.ui.rxEnvironment'])
-/**
- * @ngdoc service
- * @name rxAppRoutes.service:urlUtils
- * @description
- *
- * Set of utility functions used by rxAppRoutes to break apart/compare URLs
- */
-.service('urlUtils', function ($location, rxEnvironmentUrlFilter, $interpolate, $route, $document) {
```
Comment (CITguy) [inline src/rxAppRoutes/scripts/rxAppRoutes.js]: moved to `src/rxAppRoutes/scripts/urlUtils.js`

File: src/rxAppRoutes/scripts/rxAppRoutes.spec.js [hunk 2]
```diff
@@ -393,182 +394,3 @@ describe('rxAppRoutes', function () {
         });
     });
 });
-
-describe('urlUtils', function () {
```
Comment (CITguy) [inline src/rxAppRoutes/scripts/rxAppRoutes.spec.js]: moved to `src/rxAppRoutes/scripts/urlUtils.spec.js`

**Comment by bry4n** [comment]
It seems Leeroy Jenkins is happy with this change. I'll approve this. LGTM :smile:

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**